llvm/test/Transforms/LoopVectorize/ARM/mve-known-trip-count.ll

```diff
@@ -195,197 +195,6 @@ for.body:                                         ; preds = %entry, %for.body
   br i1 %cmp, label %for.body, label %for.cond.cleanup
 }
 
-; Trip count of 8 - does get vectorized
```

Comment:
Why has this been deleted?

Reply:
The register pressure was too high for it to be vectorised and I couldn't modify the test to reduce the pressure and still have it vectorise at the expected VF with the same trip count. I'm now trying to find a way to modify the pressure calculation to keep it and vectorise at VF 4.

Reply:
I've managed to reduce the test so that it doesn't exhibit huge register pressure, do let me know what you think of it.